bgpd/bgp_zebra.c

```diff
@@ -1781,8 +1781,13 @@ static void bgp_handle_route_announcements_to_zebra(struct event *e)
 			break;
 
 		table = bgp_dest_table(dest);
+		/*
+		 * Table cannot be NULL.
+		 * This logic of new zebra list is not applicable during cleanup.
+		 */
+		assert(table);
```

Comment:
I would just remove the assert and remove the test for table being a useable pointer on line 1790 below.

Reply:
Makes sense. Will do